drivers/wifi/nrfwifi/CMakeLists.txt

```diff
@@ -158,6 +162,16 @@ endif()
 zephyr_compile_definitions(
   -DCONFIG_NRF_WIFI_FW_BIN=${NRF70_PATCH}
 )
+else()
+  message(WARNING "
+  ------------------------------------------------------------------------
+  Building only the nRF70 driver, skipping firmware patch.
+  This is only for building (CI) purposes and will not work on a real device.
+  ------------------------------------------------------------------------
+  ")
+endif()
```

Comment:
This certainly works but it would be easier to read if the check is reversed
```
if (CONFIG_NRF_WIFI_BUILD_ONLY_MODE)
  message(WARNING "
  ------------------------------------------------------------------------
  Building only the nRF70 driver, skipping firmware patch.
  This is only for building (CI) purposes and will not work on a real device.
  ------------------------------------------------------------------------
  ")
else()
...
endif()
```